internal/server.go

```diff
@@ -68,4 +68,8 @@ func Server() {
 	if err != nil {
 		panic(err)
 	}
+
```

Comment:
If the code is on static, this part is not required. Furthermore, since this is below ListenAndServe, it is not added, so please fix or remove